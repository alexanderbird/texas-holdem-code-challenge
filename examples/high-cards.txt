4H 2S 7D 6S 5D
Becky 9C
Dylan AC
Edward QC
Henry KC 
Illana JC
Julie TC
