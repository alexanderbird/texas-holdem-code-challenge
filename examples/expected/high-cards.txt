1 Dylan Highcard Ace
2 Henry Highcard King
3 Edward Highcard Queen
4 Illana Highcard Jack
5 Julie Highcard 10
6 Becky Highcard 9
